### PR TITLE
adding appropriate error msg for encrypted pdf file

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16026

## Description
Adding appropriate error msg for encrypted pdf file

### CODE REVIEW

File: Scripts/script-ReadPDFFile.yml [hunk 1]
```diff
@@ -56,6 +57,9 @@ script: |-
           except PSEOF:
               mark_suspicious()
               sys.exit(0)
+          except PDFEncryptionError as err:
```
Comment (glicht) [inline Scripts/script-ReadPDFFile.yml]: Let's handle this similar to how we handle PSEOF and not return an error. Human readable content should contain the error message as a regular entry.

Comment (ronykoz) [reply Scripts/script-ReadPDFFile.yml]: Not sure if that's the right way to go here, when we encounter PSEOF we mark the file as suspicious and return an error entry, are you sure it's the right way to go here?

**Comment by glicht** [comment]
@ronykoz Looks good. Can you just add a test with an encrypted file.

**Comment by glicht** [comment]
@ronykoz All looks good. Build failure seems unrelated. Do you want me to force merge?

**Comment by ronykoz** [comment]
@glicht yes please

**Pull request merged**